docs/logging.rst

```diff
@@ -28,8 +28,11 @@ configuring whole loggers in your application.
 Access logs
 -----------
 
-Access log by default is switched on and uses ``'aiohttp.access'``
-logger name.
+Access logs are enabled by default. If the `debug` flag is set, and the default
+logger ``'aiohttp.access'`` is used, access logs will be output to
+:obj:`~sys.stderr` if no handlers are attached.
```

Comment:
What does tilde `~` mean?

Reply:
Tildes make qualified cross-references only show the last part.